<launch>
    <!-- 定义参数，使用args从命令行传入-->
    <arg name="input_deg" default="-60.0" />
    <arg name="input_thrust" default="0.5" />

    <!-- 启动节点，传递参数 -->
    <node name="attitude_control_node" pkg="usv_run" type="003attitude_control_node" output="screen">
        <param name="input_deg" value="$(arg input_deg)" />
        <param name="input_thrust" value="$(arg input_thrust)" />
    </node>
</launch>
